src/placeos-rest-api/controllers/users.cr

```diff
@@ -48,6 +48,8 @@ module PlaceOS::Api
 
     getter user : Model::User do
       lookup = params["id"]
+
+      # NOTE:: remove after June 2023, added to help with 2022 user id migration
```

Comment:
```suggestion
      # TODO: Remove user id query prefixing.
      # Remove after June 2023, added to help with 2022 user id migration
```

This will trigger the TODO to issues helper